enricher/fabric8/src/main/java/io/fabric8/maven/enricher/fabric8/AutoTLSEnricher.java

```diff
@@ -89,7 +89,7 @@ public void visit(PodSpecBuilder builder) {
                 }
                 String jksSecretVolumeName = getConfig(Config.jksVolumeName);
                 if (!isVolumeAlreadyExists(builder.buildVolumes(), jksSecretVolumeName)) {
-                    builder.addNewVolume().withName(jksSecretVolumeName).withNewEmptyDir("Memory").endVolume();
+                    builder.addNewVolume().withName(jksSecretVolumeName).withNewEmptyDir().withMedium("Memory").endEmptyDir().endVolume();
```

Comment:
Is this change needed?

Reply:
there's no parameters on `withEmptyDir()` any more - `withMedium()` was the only place I could find a string parameter ;) no clue really what we should do with the `"Medium"` string really

Reply:
strange, in my case it's `builder.addNewVolume().withName(jksSecretVolumeName).withNewEmptyDir("Memory").endVolume();` still getting valid without any compilation error. 
I'm doing something wrong here then.

Reply:
whoah - maybe its somehting wrong with my step; I tried that and it didn't compile for me. Wanna have a go at a PR yourself? :)

Reply:
Sure